WooCommerce/src/main/kotlin/com/woocommerce/android/ui/jetpack/benefits/JetpackBenefitsScreen.kt

```diff
@@ -58,6 +62,8 @@ fun JetpackBenefitsScreen(
             .background(MaterialTheme.colors.surface)
             .fillMaxSize()
             .padding(dimensionResource(id = R.dimen.major_100))
+            .windowInsetsPadding(WindowInsets.navigationBars)
+            .windowInsetsPadding(WindowInsets.displayCutout)
```

Comment:
Minor np, you can probably use higher level modifier `safeDrawing` here instead. That'd would cover all the cases we need safely and in a single line of code. 

```suggestion
            .windowInsetsPadding(WindowInsets.safeDrawing)
```

Reply:
Done here - a424afc563c74eedf6ec4374571c8e00e584a65b